tensorflow/python/keras/benchmark/keras_examples_benchmark_test.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import six
+
+from tensorflow.python import keras
+from tensorflow.python.keras import layers
+from tensorflow.python.keras import preprocessing
+from tensorflow.python.keras.datasets import imdb
+from tensorflow.python.keras.benchmark import benchmark_util
+from tensorflow.python.platform import benchmark
+from tensorflow.python.platform import test
+
+_MAX_FEATURE = 20000
+_MAX_LEN = 200
+
+
+class KerasExamplesBenchmark(
+    six.with_metaclass(benchmark.ParameterizedBenchmark, test.Benchmark)):
+  """Benchmark for examples on https://keras.io/examples."""
+
+  """Required Arguments for measure_performance:
+      x: Input data, it could be Numpy or load from tfds.    
+      y: Target data. If `x` is a dataset, generator instance,
+         `y` should not be specified.
+      loss: Loss function for model.
+      optimizer: Optimizer for model.
+      Other details can see in `measure_performance()` method of 
+      benchmark_util.
+  """
+  # Set different batch_size and iteration as needed.
```

Comment:
nit: This is the parameter for different configs of benchmark tests.

Can you also specify the format of the parameter? Something like:

  The parameter of each benchmark is a tuple, including:
  1) The suffix of the benchmark test name with convention "{bs}_{batch_size}";
  2) The batch_size;
  3) The number of iterations.

So others can understand how this parameterized test works, and easily add new parameters if needed.